server/channels/api4/ip_filtering.go

```diff
@@ -81,7 +82,11 @@ func applyIPFilters(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec.Success()
 
-	go c.App.SendIPFiltersChangedEmail(c.AppContext, c.AppContext.Session().UserId)
+	go func() {
+		if err := c.App.SendIPFiltersChangedEmail(c.AppContext, c.AppContext.Session().UserId); err != nil {
+			c.Err = model.NewAppError("SendIPFiltersChangedEmail", "api.context.ip_filtering.send_email.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
```

Comment:
We can't return the error via HTTP as the request might have finished already. Please just log the error.